index.js

```diff
@@ -7,43 +7,12 @@ function ask(questionText) {
   });
 }
 
-function getMedian(maximum, minimum) {
-  return Math.round((maximum + minimum) / 2)
```

Comment:
Nice use of a binary search.  Did using `Math.round` instead of `Math.floor` give you any off by 1 errors?